package.json

```diff
@@ -40,15 +40,12 @@
     "markdown-link-extractor": "^4.0.2",
     "needle": "^3.3.1",
     "progress": "^2.0.3",
-    "proxy-agent": "^6.4.0",
-    "xmlbuilder2": "^3.1.1"
+    "proxy-agent": "^6.4.0"
   },
   "devDependencies": {
-    "@eslint/js": "^9.14.0",
```

Comment:
Used
https://github.com/tcort/markdown-link-check/blob/f6bc5880022461f994f51c37c824df6d5ed5c9cc/eslint.config.mjs#L1

Reply:
[It is included as part of the eslint dependencies](https://github.com/tcort/markdown-link-check/pull/415/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R1123).

Reply:
For now it it, but it's also recommended to separately install it as part of the docs https://eslint.org/docs/latest/use/getting-started#manual-set-up

Reply:
OK, I've added that back.